src/components/SQForm/SQFormReadOnlyField.tsx

```diff
@@ -42,7 +42,6 @@ function SQFormReadOnlyField({
         InputProps={{
           ...InputProps,
           readOnly: true,
-          disableUnderline: true,
```

Comment:
I assume this is a new default?

Reply:
I believe this prop is no longer a prop after upgrading. Which is why it was removed.

Reply:
fair